webui/react/src/services/decoder.ts

```diff
@@ -3,23 +3,67 @@ import { isNumber } from 'util';
 import dayjs from 'dayjs';
 
 import {
-  decode, ioAgents, ioDetailedUsers, ioDeterminedInfo, ioExperiment, ioExperimentDetails,
-  ioExperiments, ioGenericCommand, ioGenericCommands, ioLog, ioLogin, ioLogs, ioTaskLogs,
-  ioTrialDetails, ioTypeAgents, ioTypeCheckpoint, ioTypeDetailedUsers, ioTypeDeterminedInfo,
-  ioTypeExperiment, ioTypeExperimentConfig, ioTypeExperimentDetails, ioTypeExperiments,
-  ioTypeGenericCommand, ioTypeGenericCommands, ioTypeLog, ioTypeLogin, ioTypeLogs, ioTypeMetric,
-  ioTypeStep, ioTypeTaskLogs, ioTypeTrial, ioTypeTrialDetails, ioTypeValidationMetrics,
+  decode,
+  ioDetailedUsers,
+  ioDeterminedInfo,
+  ioExperiment,
+  ioExperimentDetails,
+  ioExperiments,
+  ioGenericCommand,
+  ioGenericCommands,
+  ioLog,
+  ioLogin,
+  ioLogs,
+  ioTaskLogs,
+  ioTrialDetails,
+  ioTypeCheckpoint,
+  ioTypeDetailedUsers,
+  ioTypeDeterminedInfo,
+  ioTypeExperiment,
+  ioTypeExperimentConfig,
+  ioTypeExperimentDetails,
+  ioTypeExperiments,
+  ioTypeGenericCommand,
+  ioTypeGenericCommands,
+  ioTypeLog,
+  ioTypeLogin,
+  ioTypeLogs,
+  ioTypeMetric,
+  ioTypeStep,
+  ioTypeTaskLogs,
+  ioTypeTrial,
+  ioTypeTrialDetails,
+  ioTypeValidationMetrics,
 } from 'ioTypes';
 import { waitPageUrl } from 'routes/utils';
 import {
-  Agent, Checkpoint, CheckpointState, CheckpointStorageType, Command, CommandState,
-  CommandType, DetailedUser, DeterminedInfo, ExperimentBase,
-  ExperimentConfig, ExperimentDetails, ExperimentItem, Log, LogLevel, RawJson,
-  ResourceState, ResourceType, RunState, Step, TrialDetails, TrialItem, ValidationMetrics,
+  Agent,
+  Checkpoint,
+  CheckpointState,
+  CheckpointStorageType,
+  Command,
+  CommandState,
+  CommandType,
+  DetailedUser,
+  DeterminedInfo,
+  ExperimentBase,
+  ExperimentConfig,
+  ExperimentDetails,
+  ExperimentItem,
+  Log,
+  LogLevel,
+  RawJson, ResourceState,
+  ResourceType,
+  RunState,
+  Step,
+  TrialDetails,
+  TrialItem,
+  ValidationMetrics,
 } from 'types';
 import { capitalize } from 'utils/string';
 
 import * as AB from './api-ts-sdk'; // API Bindings
+import * as Api from './api-ts-sdk';
```

Comment:
**block:** duplicate import. @hamidzr how about `Sdk` instead of `AB`? `AB` isn't very descriptive, `Api` is too common, `Sdk` provides enough uniqueness...

Reply:
Done.